src/dashboard/templates/500.html

```diff
@@ -36,20 +36,7 @@ <h1>
-            <path fill="#6A6AF4" d="M166.036996 158.982c.62-1.24 2.328-1.305 3.063-.196l.113.196 21.3 42.6c.559 1.118-.182 2.423-1.384 2.557l-.204.011h-7.1c-.98 0-1.775-.794-1.775-1.775 0-.91.685-1.66 1.568-1.763l.207-.012h4.226l-18.426-36.856-18.428 36.856h25.528c.91 0 1.661.686 1.763 1.568l.012.207c0 .911-.685 1.661-1.568 1.764l-.207.011h-28.4c-1.25 0-2.086-1.246-1.669-2.38l.081-.188 21.3-42.6Z" />
-            <path fill="#6A6AF4" d="M167.624996 175.75c.98 0 1.775.795 1.775 1.775v10.65c0 .981-.795 1.775-1.775 1.775-.98 0-1.775-.794-1.775-1.775v-10.65c0-.98.795-1.775 1.775-1.775Zm1.775 19.525c0-.98-.795-1.775-1.775-1.775-.98 0-1.775.795-1.775 1.775 0 .981.795 1.775 1.775 1.775.98 0 1.775-.794 1.775-1.775Z" />
-            <path fill="#000091" d="M190.687996 120.519c-.102-.883-.853-1.568-1.763-1.568H93.0749962l-.207.012c-.8828.102-1.568.852-1.568 1.763v81.65l.0119.207c.1026.882.8528 1.568 1.7631 1.568h42.5999998l.207-.012c.883-.103 1.568-.853 1.568-1.763l-.012-.207c-.102-.883-.853-1.568-1.763-1.568H94.8499962v-63.9h65.6749998l.207-.012c.883-.103 1.568-.853 1.568-1.763 0-.981-.795-1.775-1.775-1.775H94.8499962v-10.65h92.2999998v10.65h-15.975l-.207.012c-.883.102-1.568.852-1.568 1.763 0 .98.795 1.775 1.775 1.775h15.975v44.375l.012.207c.102.882.853 1.568 1.763 1.568.98 0 1.775-.795 1.775-1.775v-60.35l-.012-.207Z" />
-            <path fill="#000091" d="M100.174996 129.601c-.9786998 0-1.7749998-.797-1.7749998-1.775 0-.979.7963-1.775 1.7749998-1.775.979 0 1.775.796 1.775 1.775 0 .978-.796 1.775-1.775 1.775Zm5.325-1.775c0 .978.796 1.775 1.775 1.775.979 0 1.775-.797 1.775-1.775 0-.979-.796-1.775-1.775-1.775-.979 0-1.775.796-1.775 1.775Zm8.875 1.775c-.979 0-1.775-.797-1.775-1.775 0-.979.796-1.775 1.775-1.775.979 0 1.775.796 1.775 1.775 0 .978-.796 1.775-1.775 1.775Zm67.45-3.55c.98 0 1.775.794 1.775 1.775 0 .91-.685 1.66-1.568 1.763l-.207.012h-53.25c-.98 0-1.775-.795-1.775-1.775 0-.911.685-1.661 1.568-1.763l.207-.012h53.25Zm-30.175 23.075c0-.981-.795-1.775-1.775-1.775h-42.6l-.207.012c-.883.102-1.568.852-1.568 1.763 0 .98.795 1.775 1.775 1.775h42.6l.207-.012c.883-.103 1.568-.853 1.568-1.763Zm-8.875 8.875c.98 0 1.775.794 1.775 1.775 0 .91-.685 1.66-1.568 1.763l-.207.012h-35.5c-.98 0-1.775-.795-1.775-1.775 0-.911.685-1.661 1.568-1.763l.207-.012h35.5Zm-8.875 12.425c0-.981-.795-1.775-1.775-1.775h-24.85l-.207.012c-.883.102-1.568.852-1.568 1.763 0 .98.795 1.775 1.775 1.775h24.85l.207-.012c.883-.103 1.568-.853 1.568-1.763Z" />
-            </g>
-          </svg>
+          <img src="{% static "/dashboard/img/technical-error.svg" %}" width="160" height="200" alt=""/>
```

Comment:
You should provide an alt description.

Reply:
yes it is intentional, for a decorative image like this one, the alt should be empty.
- A decorative image does not need a description, because it does not provide any useful information.
- Leaving the `alt` attribute empty tells assistive tools  that they can ignore the image.